Assets/Tests/EditMode/AuthTests/TokenSaving/PlayerPrefsStringSaverTests.cs

```diff
@@ -0,0 +1,49 @@
+using NUnit.Framework;
+using Src.Auth.AuthTokenSaver.PlayerPrefsStringSaver;
+using UnityEngine;
+
+namespace Tests.EditMode.AuthTests.TokenSaving
+{
+    [TestFixture]
+    public class PlayerPrefsStringSaverTests
+    {
+        private const string SamplePrefName = "SAMPLE_PREF";
+        private const string SamplePrefValue = "SAMPLE_PREF_VALUE";
+
+        [Test]
+        public void SaveStringValue_ShouldSaveCorrectly()
+        {
+            var saver = new StringSaver();
+
+            saver.SaveStringValue(SamplePrefName, SamplePrefValue);
+
+            var storedValue = PlayerPrefs.GetString(SamplePrefName);
+            PlayerPrefs.DeleteKey(SamplePrefName);
+
+            Assert.IsTrue(storedValue == SamplePrefValue);
+        }
+
+        [Test]
+        public void TryGetStringValue_ShouldReturnValid()
```

Comment:
Seems like name of this test isn't finished. Add what should be valid exactly and in what case should it be returned.